src/cross_module_fn.c

```diff
@@ -93,6 +92,7 @@ CROSSMODULE_WRAPPER(chunk_get_relstats);
 CROSSMODULE_WRAPPER(chunk_get_colstats);
 CROSSMODULE_WRAPPER(chunk_create_empty_table);
 CROSSMODULE_WRAPPER(chunk_create_replica_table);
+CROSSMODULE_WRAPPER(chunk_status);
```

Comment:
Almost all code that gets catalog metadata uses the Apache license. `chunk_status()` only calls `ts_chunk_get_by_relid()` which is also an Apache function. I don't think `chunk_status()` should be a TSL function.

Reply:
That makes sense. Move the function.